crates/next-core/js/src/dev/hmr-client.ts

```diff
@@ -20,7 +20,10 @@ import {
 } from "../overlay/client";
 import { addEventListener, sendMessage } from "./websocket";
 import { ModuleId } from "@vercel/turbopack-runtime/types";
-import { HmrUpdateEntry } from "@vercel/turbopack-runtime/types/protocol";
+import {
```

Comment:
you can use `import type` for type only imports. That's won't lead to an import at runtime so it's cheaper